Jenkinsfile

```diff
@@ -31,7 +31,7 @@ pipeline {
     stage('Default Docker Stages') {
       agent {
         docker {
-          image 'kieker/kieker-build:openjdk8'
+          image 'kieker/kieker-build:openjdk11'
```

Comment:
Building everything with JDK 11 will lead to Bytebuffer problems, making also kieker-monitoring effectively unusable with JDK 8 (https://stackoverflow.com/a/61267496/2096209). Therefore, for keeping compatibility of monitoring with JDK, and at the same time compile analysis with JDK 11, we need to separate JDKs.

Reply:
We cannot compile most of the analysis when no jdk11 is available. More and more libraries are only compiled for Java11. However, in build.gradle we have set code source and target compatibility for kieker-monitoring to Java8. Thus, it should still be usable with Java8.

Reply:
Also there is no API change between Kieker 1.15.x and 2.0.0 in monitoring. Thus, in case of monitoring Java8 is necessary, you can always use older Kieker versions.